install.sh

```diff
@@ -546,18 +546,7 @@ getshims() {
 killtree $({ set +x; } 2>/dev/null; getshims; set -x)
 
 do_unmount() {
-    { set +x; } 2>/dev/null
-    MOUNTS=
-    while read ignore mount ignore; do
-        MOUNTS="$mount\n$MOUNTS"
-    done </proc/self/mounts
-    MOUNTS=$(printf $MOUNTS | grep "^$1" | sort -r)
-    if [ -n "${MOUNTS}" ]; then
-        set -x
-        xargs -n 1 umount <<< ${MOUNTS}
-    else
-        set -x
-    fi
+    awk -v path="$1" '$2 ~ ("^" path) { print $2 }' /proc/self/mounts | sort -r | xargs -t -n 1 umount
```

Comment:
Just ensure you test this script against busybox awk.

Reply:
```
brandond@dev01:/var/lib/repos/go/src/github.com/rancher/k3s$ docker run --rm -it busybox
/ # awk 2>&1 | grep UTC
BusyBox v1.32.0 (2020-09-08 18:09:07 UTC) multi-call binary.
/ # awk -v path="/var" '$2 ~ ("^" path) { print $2 }' /proc/self/mounts | sort -r | xargs -r -t -n 1 echo umount
/ # awk -v path="/etc" '$2 ~ ("^" path) { print $2 }' /proc/self/mounts | sort -r | xargs -r -t -n 1 echo umount
echo umount /etc/resolv.conf
umount /etc/resolv.conf
echo umount /etc/hosts
umount /etc/hosts
echo umount /etc/hostname
umount /etc/hostname
```